ocs_ci/resiliency/resiliency_tools.py

```diff
@@ -106,3 +106,56 @@ def is_ceph_health_ok(self):
             return True
         log.error("Ceph cluster is not healthy.")
         return False
+
+    def get_ceph_health(self, detail=False):
```

Comment:
Do we need to redefine similar functions definition here? Ref: cluster.py https://github.com/red-hat-storage/ocs-ci/blob/7d156bbd2e31ab34f87c688b1623a05bae53841e/ocs_ci/ocs/cluster.py#L610